automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimePatchSpec.scala

```diff
@@ -19,14 +19,13 @@ import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 import org.broadinstitute.dsde.workbench.service.util.Tags
 import org.http4s.headers.Authorization
 import org.http4s.{AuthScheme, Credentials}
-import org.scalatest.{DoNotDiscover, Outcome}
-import org.scalatest.freespec.FixtureAnyFreeSpec
+import org.scalatest.DoNotDiscover
 
 import scala.concurrent.duration._
 
 // extending fixture.Freespec just so we can use `taggedAs`. Not really need fixture for anything
 @DoNotDiscover
-class RuntimePatchSpec extends FixtureAnyFreeSpec with LeonardoTestUtils with LeonardoTestSuite with NotebookTestUtils {
+class RuntimePatchSpec extends GPAllocFixtureSpec with LeonardoTestUtils with LeonardoTestSuite with NotebookTestUtils {
```

Comment:
Probably can now define the tests like 
```
"Patch endpoint should perform a stop/start tranition for GCE VM" taggedAs Tags.SmokeTest in { googleProject =>
 …
```

and remove the lines that read the system property and construct a `GoogleProject`.